src/RootCloseWrapper.js

```diff
@@ -26,7 +26,9 @@ export default class RootCloseWrapper extends React.Component {
     // If the click originated from within this component, don't do anything.
     // e.srcElement is required for IE8 as e.target is undefined
     let target = e.target || e.srcElement;
-    if (domUtils.contains(React.findDOMNode(this), target)) {
```

Comment:
I don't know - I feel like if you have `DropdownButton.Title`, then having `DropdownButtonTitle` as a separate import-able module rather than an implementation detail is a little bit redundant.

In an ES6 module world, though, I think it'd be odd not to write this as

```js
// Modal.js
export {default as Title} from './impl/ModalTitle';
```

which would have you write

```js
// ModalUser.js
import {Modal, Title as ModalTitle} from 'react-bootstrap/lib/Modal';
```

Or maybe having them be static properties of the `Modal` class is fine.